### PR TITLE
Ignore prefixes

### LINKED ISSUE #87: Ignore messages in certain channels or with certain prefixes
Per suggestion of @EarlJr53, Jay, and Bri

When there are game bots or whatnot in the server, people may gain levels quicker than normal if they use them often. We should add a way to blacklist certain channels or message prefixes from counting towards levels.
E.g.
`config.json`
```json
{
  "levels": {
    "ignoreChannels": ["channel1", "channel2"],
    "ignorePrefix": ["!", "m!", "m.", "p!"]
  }
}
```

### PR DESCRIPTION
**Pull request by EarlJr53**
Ignores bot command prefixes when giving out XP. Closes #87 

I decided for now that we only need to ignore prefixes, not channels.

### CODE REVIEW

**Comment by samollari** [comment]
Is this ready for review?

**Review by samollari — COMMENTED** [review]
Looks good to me but it won't merge until there's another commit that has tests run on it. So commit some minor change that won't be formatted by Prettier and I can approve it and merge

**Comment by EarlJr53** [comment]
Should be ready for review, yes. I can go find some minor change to make.

**Comment by samollari** [comment]
u g h. i'll approve but i have to re-enable override

**Review by samollari — DISMISSED** [review]

File: config.json.example [hunk 1]
```diff
@@ -18,6 +18,16 @@
...
         ],
+        "ignorePrefix": [
+            "!", 
+            "m!", 
+            "m.", 
+            "p!", 
+            "//",
+            "i-",
+            ",",
+            "git"
```
Comment (Muirrum) [inline config.json.example]: The example config should not have any defined values in beyond one or two examples.

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Review by Muirrum — APPROVED** [review]

**Pull request merged**